src/misc/setup.cpp

```diff
@@ -99,6 +99,12 @@ bool Value::operator==(const Value& other) const
 	return false;
 }
 
+bool Value::operator<(const Value& other) const
+{
+	return std::tie(_hex, _bool, _int, _string, _double) <
```

Comment:
Brilliant usage of `std::tie` :)

Reply:
Every struct and class is close to just being a tuple! One `tie` away :-)